_i18n/es/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,147 +1,114 @@
-private key to a party, that party can tell whether that transaction sent monero to that
-particular address. Note that storing these per-transaction keys is disabled by default, and
-you will have to enable it before sending, if you think you may need it:
+Si pagas a un comerciante, y el comerciante reclama que no ha recibido el pago, puedes necesitar probar a un tercero que sí enviaste los fondos, o incluso al comerciante, si es que es un error honesto. Monero es privado, así que no puedes simplemente indicar tu transacción en la blockchain, tampoco puedes saber quién la envió, ni quién la recibió. No obstante, proveyendo la llave privada por transacción a un equipo, ese equipo puede saber si esa transacción envió Monero a esa dirección en particular. Ten en cuenta que guardar estas llaves privadas por transacción está desactivado por defecto, y tendrás que activarlo antes de enviar, si crees que lo puedes necesitar:
 
     set store-tx-info 1
 
-You can retrieve the tx key from an earlier transaction:
+Puedes recuperar la llave tx de una transacción anterior:
 
     get_tx_key 1234567890123456789012345678901212345678901234567890123456789012
 
-Pass in the transaction ID you want the key for. Remember that a payment might have been
-split in more than one transaction, so you may need several keys. You can then send that key,
-or these keys, to whoever you want to provide proof of your transaction, along with the
-transaction id and the address you sent to. Note that this third party, if knowing your
-own address, will be able to see how much change was returned to you as well.
+Coloca el ID de la transacción del cual quieres la llave. Recuerda que un pago debe haber sido dividido en más de una transacción, así que puedes necesitar varias llaves. Puedes enviar esa llave, o llaves, a quien quieras proveer con pruebas de tu transacción, junto con los ID de transacción y la dirección a la que enviaste. Ten en cuenta que este tercero, si conoce tu dirección, será capaz de ver cuánto cambio regresó a tu cuenta también.
 
-If you are the third party (that is, someone wants to prove to you that they sent monero
-to an address), then you can check this way:
+Si tú eres el tercero (esto es, alguien quiere probarte que enviaron Monero a una dirección), entonces puedes revisarlo de esta forma:
 
     check_tx_key TXID TXKEY ADDRESS
 
-Replace `TXID`, `TXKEY` and `ADDRESS` with the transaction ID, per-transaction key, and destination
-address which were supplied to you, respectively. monero-wallet-cli will check that transaction
-and let you know how much monero this transaction paid to the given address.
+Reemplaza `TXID`, `TXKEY` y `ADDRESS` con el ID de transacción, la llave por transacción y la dirección de destino que te fueron provistas respectivamente. monero-wallet-cli revisará esa transacción y te hará saber cuánto monero se utilizó en esta transacción a la dirección dada.
```

Comment:
I think the word "paid" should stay as "pago". It's a payment made to an address. The word "utilizar/ utilizó" refers to "how much", it can be confused with, "how much monero" was used in the transaction, including fees? Is a little strange when you read it.